Source/WebCore/Modules/mediastream/RTCStatsReport.h

```diff
@@ -61,26 +87,44 @@ class RTCStatsReport : public RefCounted<RTCStatsReport> {
         Certificate,
     };
     struct Stats {
+        Stats() = default;
+#if USE(LIBWEBRTC)
+        Stats(Type, const webrtc::RTCStats&);
+#endif
+
         double timestamp { 0 };
         Type type;
```

Comment:
Shouldn't this be initiises too?

Reply:
The constructor should do it

Reply:
Plan is to remove the default constructor so that Stats(Type,...) is responsible for it.